tests/valid/CompoundMultDivWithDifferentButCompatibleUnits.nestml

```diff
@@ -0,0 +1,35 @@
+"""
+CompoundMultDivWithDifferentButCompatibleUnits.nestml
+########################################################
+
+
+Copyright statement
++++++++++++++++++++
+
+This file is part of NEST.
+
+Copyright (C) 2004 The NEST Initiative
+
+NEST is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+NEST is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+"""
+neuron MultDivTest:
+    state:
+        lhs V = 1 V
+    end
+
+    update:
+        lhs *= 1200mV
```

Comment:
Is this a valid use case? The lhs is in `V` and the rhs is in `V*mV`

Reply:
This turned out to be an issue with test discovery, in particular, these compound operator .nestml files were not being processed by ``cocos_test.py``. This was fixed by 747bae7.